the-elder-chores-user-ui/src/pages/Login/Login.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { useNavigate } from "react-router-dom";
+import { LoginForm } from "@patternfly/react-core";
+import './Login.css';
+import { usePocketbase } from '../../components/Pocketbase';
+
+const Login = () => {
+
+  const client = usePocketbase();
+  const navigate = useNavigate();
+
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [showHelperText, setShowHelperText] = useState(false);
+  const [isValidUsername, setIsValidUsername] = useState(true);
+  const [isValidPassword, setIsValidPassword] = useState(true);
+
+  const handleUsernameChange = (value) => {
+    setUsername(value);
+  };
+
+  const handlePasswordChange = (value) => {
+    setPassword(value);
+  };
+
+  const handleSubmit = async event => {
+    event.preventDefault();
+    setIsValidUsername(!!username);
+    setIsValidPassword(password.length >= 10);
+    setShowHelperText(!isValidUsername || !isValidPassword);
+    await client.users.authViaEmail(username, password);
+    navigate("/");
+  }
+
+  return(
+    <LoginForm
+      showHelperText={showHelperText}
+      helperText="Invalid login credentials. Username can't be blank. Password must have at least 10 characters."
+      usernameLabel="Username"
+      usernameValue={username}
+      onChangeUsername={handleUsernameChange}
+      isValidUsername={isValidUsername}
+      passwordLabel="Password"
+      passwordValue={password}
+      onChangePassword={handlePasswordChange}
+      isValidPassword={isValidPassword}
+      onLoginButtonClick={handleSubmit}
```

Comment:
@josejulio any idea on how to get the user automatically redirected to the home page after login?

Reply:
Since react doesn't detect the change in the state, we need to "force it" with 
```
const history = useHistory();  // from react-router

history.go(0); // when we are ready to refresh, call this
```

Reply:
Turns out useHistory is from v5 and useNavigate is from v6, which is the one we're using.

Reply:
mmh i see- maybe using the {replace:true} params?